src/renderer/components/ft-community-poll/ft-community-poll.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+  <div class="poll">
+    <div
+      class="vote-count"
+    >
+      <!-- Format the votes to be split by commas ie. 1000 -> 1,000 -->
+      {{ `${data.totalVotes.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ",")} votes` }}
```

Comment:
If you switch to the `parseLocalSubscriberCount` in the other file (see other comment), you don't have to do any replace stuff here.

Additionally please add a translation key for the votes string, ideally with a place holder e.g. `{votes} votes`. It will also need to be added to the `static/locales/en-US.yaml` file.

Here is an example of a translation string with a placeholder:
https://github.com/FreeTubeApp/FreeTube/blob/aa1d422210f0951068e0405042d38509fcf2e525/src/renderer/components/general-settings/general-settings.vue#L119

Reply:
This was also just to make the vote count pretty via comma separation so `1000000` -> `1,000,000` but if this looks too convoluted I can toss it.

Reply:
Please don't hard code it, as other languages don't separate by 3 digits and use other separator characters.

If you want separators, add a computed property for the totalVotes (e.g. totalVotesFormattted), import and use the `formatNumber` helper function from the utils.js file, which formats the number with the currently selected app display language.